assets/css/style.css

```diff
@@ -1,3 +1,276 @@
-.makeHeaderRed {
-  color: red;
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: calc(20px + 0.390625vw);
+}
+
+body {
+  background-color: black;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 18px;
+  background-color: #3e3c3c;
+  box-shadow: 1px 2px 10px black;
+}
+
+.logo {
+  margin-left: 3%;
+}
+
+.hamburgerMenu {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: center;
+}
+
+.nav-items {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+  padding-right: 30%;
+}
+
+.list-item {
+  list-style: none;
+  margin: 0;
+  padding: 0 25px;
+}
+
+.list-item a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.list-item a:hover,
+button:hover {
+  transform: translate(4px, 2px);
+  box-shadow: #ff6b00;
+}
+
+.hamburger-icon {
+  display: none;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  margin-right: 20px;
+}
+
+.hamburger-icon .bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #e5e5e5;
+  border-radius: 2px;
+}
+
+.main-area {
+  background-image: url("../images/Illustration-header.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+}
```

Comment:
- [ ] Kindly adjust the `background-position` used for this section so that it looks identical to that of the Figma template that you chose. 

### Your Design

![Screenshot 2023-02-18 005301](https://user-images.githubusercontent.com/77106962/219818126-e7fdb93a-574e-41f7-93d3-e5d5141ffcf6.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/219818205-62dfff50-d820-4a55-9bcc-318dcee02c65.png)


#### Hint: Kindly notice the space that I marked with orange ink. That space shouldn't be there. Kindly adjust it. 👍 💻

Reply:
@Gambit142 Thank you! Your feedback was very helpful. Have an amazing day.